react/features/rtcstats/middleware.any.ts

```diff
@@ -176,6 +138,12 @@ MiddlewareRegistry.register(store => next => action => {
         }
         break;
     }
+    case CONFERENCE_WILL_LEAVE: {
```

Comment:
How well does this work in the context of breakout rooms? On mobile we switch between rooms without a reload.

Reply:
Tested it now. On web, it will end the stats session when leaving the main conference and create a new one when joining back, without creating a new session in the breakout room. On mobile, it will create a new stats session whenever switching the room. I'm not sure what is the right behavior.

Reply:
Can you please sync with George / Nils / Andrei to clarify what the right behavior should be?